docs/source/etcd-data-model.rst

```diff
@@ -117,6 +132,13 @@ The various properties in this object have the following meanings:
 
   .. note:: Currently only /32 subnets are supported.
 
+``ipv6_nat``
+  a list of 1:1 NAT mappings to apply to the endpoint.  Inbound connections to
+  out_ip will be forwarded to in_ip.  Connections initiated from in_ip will
+  not have their source address changed, except when an endpoint attempts to
+  connect one if it's own out_ips.  Each in_ip must be associated with the
```

Comment:
This needs the same correction as the `ipv4_nat` key, and I think it's meant to be

> connect *to* one of *its* own out_ips